app/src/main/java/com/github/se/eventradar/ui/navigation/NavGraph.kt

```diff
@@ -46,6 +47,14 @@ fun NavGraph(
         }
 
     // TODO replace the Toast message with the corresponding screen function of the route
+
+    composable(
+        "${Route.MY_EVENT}/{eventId}",
+        arguments = listOf(navArgument("eventId") { type = NavType.StringType })) {
+          val eventId = it.arguments!!.getString("eventId")!!
+          val viewModel = ScanTicketQrViewModel.create(eventId = eventId)
+
```

Comment:
You have to call the actual composable function...

Reply:
I deleted it from VM PR, only gonna put in UI PR :) thanks